ci-operator/config/openshift/verification-tests/openshift-verification-tests-main__installer-rehearse-4.20.yaml

```diff
@@ -67,6 +67,41 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-ibmcloud-ipi-sno
+- as: aws-ipi-custom-dns
+  cron: '@yearly'
+  steps:
+    cluster_profile: aws-qe
+    env:
+      AWS_INSTALL_USE_MINIMAL_PERMISSIONS: "yes"
+      BASE_DOMAIN: qe.gcp.devcluster.openshift.com
+      FEATURE_GATES: '["AWSClusterHostedDNS=true"]'
```

Comment:
Please see https://github.com/openshift/api/pull/2423. Names of the feature gates are being changed.

Reply:
@sadasu yes, I noticed that new name, but I think this PR could be merged very soon and jobs are available for the recent test (before new name is introduced). It's easy to update to new name after 2423 merged.

Reply:
https://github.com/openshift/api/pull/2423 has been merged, now `AWSClusterHostedDNSInstall` is used.